app/views/static_pages/styles.html.erb

```diff
@@ -0,0 +1,78 @@
+<h1>Basic Elements</h1>
+
+<h2>Button States</h2>
+<h2>Button Sizes</h2>
+<h2>Button Variants</h2>
+<h2>Social Buttons</h2>
+
+<h2>Dropdowns</h2>
+
+<h2>Inputs</h2>
+<h2>Input Variants</h2>
+
+<h2>Textarea</h2>
+
+<h2>Tags</h2>
+
+<h2>Labels</h2>
+
+<h2>Checkboxes &amp; Radio-buttons</h2>
+
+<h2> Iconography </h2>
+<h3> Social </h3>
+<i class='icon-behance'></i>
+<i class='icon-dribbble'></i>
+<i class='icon-facebook'></i>
+<i class='icon-github'></i>
+<i class='icon-google'></i>
+<i class='icon-icloud'></i>
+<i class='icon-instagram'></i>
+<i class='icon-linkedin'></i>
+<i class='icon-paypal'></i>
+<i class='icon-pinterest'></i>
+<i class='icon-skype'></i>
+<i class='icon-twitter'></i>
+
+<h3>Arrows</h3>
+<i class=''></i>
+
+<h2>Navigation</h2>
+<h2>Nav Lists</h2>
+<h2>Top Menus</h2>
+<h2>Bottom Menus</h2>
+
+<h2>Progress bars &amp; Sliders</h2>
+
+<h2>Tooltips</h2>
+
+<h2>Iconic Lists</h2>
+
+<h2>Alerts</h2>
+
+  <div class="alert alert-success">
+    <strong>Success!</strong> Write your success message here.
+    <a href="#" class="close-alert">&times</a>
```

Comment:
Technically, HTML entities would be written ending in a colon.  So this should be `&times;`.  Some browsers let you get away with forgetting the colon, some don't.  It's best to include it for that reason.